assets/pages/hello-world/helloWorld.js

```diff
@@ -0,0 +1,11 @@
+
+"use strict";
```

Comment:
By default, ES6 modules are [http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code](always in strict mode)